tests/pytorch/test_nn.py

```diff
@@ -1801,3 +1801,25 @@ def test_BiasedMultiheadAttention(feat_size, num_heads, bias, attn_bias_type, at
     out = net(ndata, attn_bias, attn_mask)
 
     assert out.shape == (16, 100, feat_size)
+
+@pytest.mark.parametrize('feat_size', [128, 512])
+@pytest.mark.parametrize('num_heads', [8, 16])
+@pytest.mark.parametrize('bias', [True, False])
+@pytest.mark.parametrize('attn_bias_type', ['add', 'mul'])
+@pytest.mark.parametrize('attn_drop', [0.1, 0.])
```

Comment:
I think we can use a single value for all the options other than `attn_bias_type` to save test resources. Otherwise, this induces 2^5 tests.